src/main/scala/io/iohk/ethereum/network/p2p/messages/PV63.scala

```diff
@@ -74,19 +75,24 @@ object PV63 {
     implicit class MptNodeEnc(obj: MptNode) extends RLPSerializable {
```

Comment:
Isn't this encoder the same as this one https://github.com/input-output-hk/mantis/pull/291/files#diff-081f56715a2b593d7116a59f2b2a251eR80 ?